docs/publishing/retraction.md

```diff
@@ -0,0 +1,24 @@
+# Retracting a nanopublication
+A nanopublication is persistent, you can never edit nor delete it.
+You can however retract a nanopublication.
+This is done by publishing a new nanopublication that states that you
+retract the original publication. You can use `NanopubClient.retract()`:
+```python
+>>> from nanopub import NanopubClient
+>>> # Create the client (we use use_test_server=True to point to the test server)
+>>> client = NanopubClient(use_test_server=True)
+>>> client.retract('http://purl.org/np/RAcTwW_JdTqZJTXA4jUTFUj6ut8OI9AqaPb1IBPsLsy-o')
+Published to http://purl.org/np/RAOLg94DSpgPVifZZvhwkJ6NBl558pVXaclrgEYhLpfoI
+```
+View the full retraction nanopublication [here](http://test-server.nanopubs.lod.labs.vu.nl/RAOLg94DSpgPVifZZvhwkJ6NBl558pVXaclrgEYhLpfoI).
```

Comment:
Note that the test server is liable to be cleared every so often, so you can't rely on test URIs to still be there in future. We may wish to also publish this test retraction to the real servers if you want to point to it here.

Reply:
This is a really good point! I now use production server nanopublications as examples everywhere. (but still having `use_test_server` in the code)